.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,194 @@
+      - name: Comment PR with E2E test results
+        if: always() && github.event_name == 'pull_request'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const totalTests = '${{ steps.analyze.outputs.total_tests }}';
+            const passedTests = '${{ steps.analyze.outputs.passed_tests }}';
+            const failedTests = '${{ steps.analyze.outputs.failed_tests }}';
+            const successRate = '${{ steps.analyze.outputs.success_rate }}';
+
+            let comment = '## 🎭 E2E Test Results\n\n';
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)